src/main.js

```diff
@@ -0,0 +1,23 @@
+function convertItemsToUnorderedList(listItems) {
+  const ulList = document.createElement("ul");
+  listItems.map(a => {
+    const liItem = document.createElement("li");
+    liItem.textContent = a.name;
+    ulList.appendChild(liItem);
+  });
+  return ulList;
+}
+
+function alphabeticCompare(a, b) {
+  return a.name > b.name ? 1 : b.name > a.name ? -1 : 0;
+}
+
+function renderInput(input, container) {
+  const dirItem = input.filter(inputEl => inputEl.type === "directory"),
```

Comment:
Since this constant is an array of values, we better rename it to plural form (e.g. `dirItems`).